ouroboros-consensus-cardano/src/unstable-cardano-tools/Cardano/Tools/DBAnalyser/Analysis/BenchmarkLedgerOps/Metadata.hs

```diff
@@ -42,7 +42,7 @@ data Metadata = Metadata {
 instance ToJSON Metadata where
   toEncoding = Aeson.genericToEncoding Aeson.defaultOptions
 
-getMetadata :: LedgerApplicationMode -> IO Metadata
+getMetadata :: ApplyOpts -> IO Metadata
```

Comment:
Looks like this only uses `ValidationOpts`, no the full `ApplyOpts`.

Is that intended?